api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -67,7 +68,7 @@
     /**
      * Configure the string value to a specified type
      * @param value the string representation of a property value
-     * @return the converted value
+     * @return the converted value or null if value is null
```

Comment:
ah. I see what you meant. I would rather put the null if value is null on the spec not this javadoc as this javadoc is applicable for all converters not just the builtin converters.